src/mpyl/utilities/http_client/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""Generic Http Client for handling requests"""
+
+import requests
+from requests.exceptions import RequestException, HTTPError
+from .exceptions import AuthorizationError, HTTPRequestError
+
+
+class HttpClient:
```

Comment:
there wasn't a build in package or something from Python that already does this?